pkg/network/apis/network/validation/validation.go

```diff
@@ -123,31 +115,6 @@ func ValidateClusterNetwork(clusterNet *networkapi.ClusterNetwork) field.ErrorLi
 		}
 	}
 
-	if clusterNet.Name == networkapi.ClusterNetworkDefault && defaultClusterNetwork != nil {
```

Comment:
what happened with this?

Reply:
> what happened with this?

It was in my comment.  Turns out in the apiserver it was never set, so this value never matterd